lib/smart_app_launch/discovery_stu1_group.rb

```diff
@@ -116,18 +116,31 @@ class DiscoverySTU1Group < Inferno::TestGroup
         in the table below to app developers. The server SHALL use both a FHIR
         CapabilityStatement and A Well-Known Uris JSON file.
       )
+        
+      input :well_known_management_url,
```

Comment:
Please maintain the original order.

Reply:
Right, these got out of order because I had originally pulled out the inputs I knew had to be optional but forgot to rearrange them after making them all optional. Fixed